Source: written by Pierre Lejeune, from Exherbo
Upstream: submitted, https://sourceforge.net/tracker/?func=detail&aid=3451702&group_id=52781&atid=468023
Reason: Fix --enable-{ipv6,gnutls} invocation
--- a/configure.in	2011-11-19 19:34:17.000000000 +0100
+++ b/configure.in	2011-12-05 21:04:02.455580787 +0100
@@ -185,8 +185,8 @@
 AC_ARG_ENABLE(ipv6,
               [AC_HELP_STRING([--enable-ipv6],
                               [compile library with IPv6 support])],
-              [with_ipv6="yes"],
-              [with_ipv6="$disable_ipv6"])
+              [with_ipv6="$enable_ipv6"],
+              [with_ipv6="no"])
 
 AC_MSG_CHECKING(for enable ipv6 in library)
 if test "x$with_ipv6" = "xyes"; then
@@ -214,8 +214,8 @@
 AC_ARG_ENABLE(gnutls,
               [AC_HELP_STRING([--enable-gnutls],
                               [build with GNUTLS SSL/TLS])],
-              [with_gnutls="yes"],
-              [with_gnutls="$disable_gnutls"])
+              [with_gnutls="$enable_gnutls"],
+              [with_gnutls="no"])
 
 AC_MSG_CHECKING(for disable openssl in library)
 if test "x$with_openssl" = "xyes"; then
